charts/sn-platform/templates/broker/function-worker-configfile-configmap.yaml

```diff
@@ -30,32 +30,33 @@ metadata:
 data:
   functions_worker.yml: |
     # Function package management
-    {{- if and .Values.tls.enabled .Values.tls.broker.enabled }}
+    workerPort: {{ .Values.functions.ports.http }}
```

Comment:
is this compatible with non-istio based tls set up?

Reply:
The TLS part configuration is not related to Istio, so it can work under non-istio env.

Reply:
And previous worker template shouldn't work well with TLS mode; For example, `workerPort` is a required configuration in the current Pulsar version.